client/app/dedicated/server/translations/Messages_fr_FR.xml

```diff
@@ -1202,4 +1202,36 @@
    <translation id="server_rendezvous_profit" qtlid="521379">Profitez-en !</translation>
    <translation id="server_rendezvous_book" qtlid="521392">Programmer un rendez-vous</translation>
    <translation id="server_rendezvous_stop_bother" qtlid="521405">Ne plus montrer</translation>
+
+   <translation id="server_datacenter_ERI_1">Londres (ERI1) - Angleterre</translation>
```

Comment:
Looks like you could have only translations for the city and the country and then build the text to display: 
```js
`${cityName} (${datacenterId.toUpperCase()}) - ${countryName}`
```

This way, it would be easier to take into account new datacenters :)

Reply:
Note that this data is displayed in multiple parts accross the manager so we could make a component that displays this data and use it everywhere

Reply:
We don't know cityName & countryName. Impossible de to concat this kind of string